src/Features/Traits/Eloquent.php

```diff
@@ -10,17 +10,151 @@
 trait Eloquent
 {
     /**
-     * @Given there is :count :model object in database
-     * @Given there should be :count :model object in database
-     * @Given there are :count :model objects in database
-     * @Given there should be :count :model objects in database
+     * @Given there is a model :model in the database
+     * @Given there is a :model in the database
```

Comment:
Maybe table with parameters that you could provide to factory's `create()` method would be handy?